app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java

```diff
@@ -19,7 +19,7 @@ public MapPlayerListAdapter(Activity context, ArrayList<Player> playerList) {
     }
 
     public View getView(int position, View view, ViewGroup parent) {
-        view = LayoutInflater.from(getContext()).inflate(R.layout.leaderboard_item_layout, null, true);
+        view = LayoutInflater.from(getContext()).inflate(R.layout.leaderboard_item_layout, null, false);
         Player player = getItem(position);
```

Comment:
Is this the focusable parameter? The one you set from true to false?

Reply:
No, it is the attachToRoot parameter. If the root is null, it is better to set it to false.

Reply:
Ok, I see, good catch!